### DISCUSSION TITLE
Should VestingScheduleRelativeTrigger Support Years-Based Vesting Periods?

### DISCUSSION THREAD

**Discussion by JSv4** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#257] [Suggestions] [answered]
The `period` field of [VestingScheduleRelativeTrigger](https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/blob/main/docs/schema/types/vesting/VestingScheduleRelativeTrigger.md) only supports months and days as time periods. Since we also have years defined elsewhere, I think it would make sense to create a `VestingPeriodInYears` type, just as we have a `VestingPeriodInDays` and `VestingPeriodInMonths`. Understood that it's certainly possible to get away without it, but seems like it would be more natural to have the option.

**Comment by pjohnmeyer** [comment, marked as answer]
We deliberately chose not to support years because "ideal" years can be expressed in days, and calendar years can be expressed in months. We should add a note to the documentation to make it clearer.
